requirements/chart-tests.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.10
+# This file is autogenerated by pip-compile with Python 3.9
```

Comment:
Is this change needed as part of this PR? It seems like it's not. It would be great to take this requirements change out of the PR.

Reply:
reverted 👍

Reply:
wasn't required for the change, just got caught up in my updates so I could run tests locally